### ISSUE TITLE
Fix: Inaccessible theme toggle button

### ISSUE THREAD

**Issue by Builder256** [frostybee/starlight-theme-galaxy#13]
Thank you for creating this ultimate cool and modern theme.

I would like to report an accessibility issue.

## Overview
There is no accessible name in the theme switcher button, which is located in the header.
People using a screen reader cannot recognize the function of this button.

---

I apologize for my poor English as I am Japanese.
Thank you for your consideration!
